packages/editor/css-properties/fetchCssPropertyKeywords.ts

```diff
@@ -13,6 +14,23 @@ const GLOBAL_KEYWORDS = [
 const fixKnownSyntaxErrors = (syntax: string) =>
   syntax.replaceAll('<image ', '<image> ')
 
+const COMMON_KEYWORDS: Record<string, string[]> = {
```

Comment:
Consider making this partial to instruct Typescript that we don't necessarily expect to find a match. Alternatively, you can set `"noUncheckedIndexedAccess": true,` in the tsconfig file for the editor/ package